composer.json

```diff
@@ -41,3 +40,4 @@
     ]
   }
 }
+
```

Comment:
The `composer.lock` has been included in this PR. Since `/composer.lock` is in the `develop` `.gitignore`, it suggests that you've not pulled `origin/develop` into your local `develop` branch before branching from it.